godtools/App/Services/Renderer/Views/MobileContent/Views/ContentButton/MobileContentButtonView.swift

```diff
@@ -201,7 +201,10 @@ class MobileContentButtonView: MobileContentView {
             super.sendEventsToAllViews(eventIds: viewModel.buttonEvents, rendererState: viewModel.rendererState)
         
         case .url:
```

Comment:
you should actually be moving away form using the button type, you should treat a button just like any other `Clickable` object. Trigger any content events attached to it and open any url defined.

Reply:
Hey @frett that sounds good!  Let's plan that for next week Monday.  I'm going to be out tomorrow and rest of day today.